apps/cosmo/cosmo_monitor.py

```
@@ -574,11 +572,11 @@ def _get_cleaning_period(self) -> tuple[datetime, datetime]:
 
     def log_cleaning_time(
         self,
-        entity: Literal["sensor.cosmo_task_status"],
-        attribute: Literal["state"],
-        old: TaskStatus,
-        new: TaskStatus,
-        kwargs: dict[str, Any],
+        entity: str,
+        attribute: str,
+        old: Any,
```

Ruff (0.15.18)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `old`

(ANN401)